web/api/routes.go

```diff
@@ -46,6 +46,18 @@ func (ar *Router) initRoutes() {
 		ar.Token(model.TokenTypeAccess, nil),
 		negroni.Wrap(ar.RequestInviteLink()),
 	)).Methods("POST")
+	auth.Path(`/{invite:invite/?}`).Handler(negroni.New(
+		ar.Token(model.TokenTypeAccess, nil),
+		negroni.Wrap(ar.GetAllInvites()),
+	)).Methods("GET")
+	auth.Path(`/invite/{id:[a-zA-Z0-9]+}`).Handler(negroni.New(
+		ar.Token(model.TokenTypeAccess, nil),
+		negroni.Wrap(ar.GetInviteByID()),
+	)).Methods("GET")
+	auth.Path(`/invite/{id:[a-zA-Z0-9]+}`).Handler(negroni.New(
+		ar.Token(model.TokenTypeAccess, nil),
```

Comment:
does it mean any user could delete invites?

Reply:
Moved to the `admin` router alongside with the `getInviteByID` endpoint